Tone/component/filter/LowpassCombFilter.ts

```diff
@@ -0,0 +1,91 @@
+import { Param } from "../../core/context/Param";
+import { InputNode, OutputNode, ToneAudioNode, ToneAudioNodeOptions } from "../../core/context/ToneAudioNode";
+import { Frequency, NormalRange, Time } from "../../core/type/Units";
+import { optionsFromArguments } from "../../core/util/Defaults";
+import { RecursivePartial } from "../../core/util/Interface";
+import { Signal } from "../../signal/Signal";
+import { FeedbackCombFilter } from "./FeedbackCombFilter";
+import { Filter } from "./Filter";
+
+interface LowpassCombFilterOptions extends ToneAudioNodeOptions {
+	delayTime: Time;
+	resonance: NormalRange;
+	dampening: Frequency;
+}
+
+/**
+ *  Tone.Lowpass is a lowpass feedback comb filter. It is similar to
+ *  Tone.FeedbackCombFilter, but includes a lowpass filter.
+ */
+export class LowpassCombFilter extends ToneAudioNode<ToneAudioNodeOptions> {
+
+	readonly name = "LowpassCombFilter";
+
+	/**
+	 * The delay node
+	 */
+	private _combFilter: FeedbackCombFilter;
+
+	/**
+	 * The lowpass filter
+	 */
+	private _lowpass: Filter;
+
+	/**
+	 * The delayTime of the comb filter.
+	 */
+	readonly delayTime: Param<Time>;
+
+	/**
+	 * The dampening control of the feedback
+	 */
+	readonly dampening: Signal<Frequency>;
+
+	/**
+	 * The amount of feedback of the delayed signal.
+	 */
+	readonly resonance: Param<NormalRange>;
+
+	input: InputNode;
+	output: OutputNode;
+
+	constructor(options?: RecursivePartial<LowpassCombFilterOptions>)
+	constructor() {
+		super(optionsFromArguments(LowpassCombFilter.getDefaults(), arguments));
+		const options = optionsFromArguments(LowpassCombFilter.getDefaults(), arguments);
+
+		this._combFilter = this.output = new FeedbackCombFilter({
```

Comment:
add `context: this.context` to the passed in object to ensure they share the same Tone.Context